src/client/components/home/style.css

```diff
@@ -0,0 +1,21 @@
+.navContainer {
```

Comment:
Include all css in a `styled-component`. I think `home` already has one, just consolidate